docs/architecture/microservices/secure-net-microservices-web-applications/index.md

```diff
@@ -70,56 +120,21 @@ Popular external authentication providers and their associated NuGet packages ar
 | **Facebook**  | **Microsoft.AspNetCore.Authentication.Facebook**         |
 | **Twitter**   | **Microsoft.AspNetCore.Authentication.Twitter**          |
 
-In all cases, the middleware is registered with a call to a registration method similar to `app.Use{ExternalProvider}Authentication` in `Startup.Configure`. These registration methods take an options object that contains an application ID and secret information (a password, for instance), as needed by the provider. External authentication providers require the application to be registered (as explained in [ASP.NET Core documentation](/aspnet/core/security/authentication/social/)) so that they can inform the user what application is requesting access to their identity.
+In all cases, you must complete an application registration procedure that is vendor dependent and that usually involves:
 
-Once the middleware is registered in `Startup.Configure`, you can prompt users to sign in from any controller action. To do this, you create an `AuthenticationProperties` object that includes the authentication provider’s name and a redirect URL. You then return a Challenge response that passes the `AuthenticationProperties` object. The following code shows an example of this.
+1. Getting a Client Application Id.
```

Comment:
```suggestion
1. Getting a Client Application ID.
```

Reply:
Is the capitalization here expected? Or should it be client application ID, etc.

Reply:
I think it's ID.